homework_1/task_03/pokemon/pokemon.go

```diff
@@ -0,0 +1,75 @@
+package pokemon
+
+import (
+	"encoding/json"
+	"github.com/pkg/errors"
+	"github.com/sethgrid/pester"
+	"io/ioutil"
+)
+
+const pokemonAPI = "https://pokeapi.co/api/v2/pokemon/"
+
+type Location struct {
+	Name string
+}
+
+type Encounter struct {
+	Location Location `json:"location_area"`
+}
+
+type Pokemon struct {
+	Name string `json:"name"`
+}
+
+type Output struct {
+	Name string
+	Locations []string
+}
+
+func GetData(url string) ([]byte, error) {
+	httpClient := pester.New()
+
+	httpResponse, err := httpClient.Get(url)
+	if err != nil {
+		return nil, errors.New("HTTP get towards pokeapi")
+	}
+
+	bodyContent, err := ioutil.ReadAll(httpResponse.Body)
+	if err != nil {
+		return nil, errors.New("error while reading pokeapi")
+	}
+
+	return bodyContent, nil
+}
+
+func FindLocations(input string) ([]byte, error) {
```

Comment:
`input` here is a very vague name. I know it's an input to a function - this is not useful information.